jekyll/faq.md

```diff
@@ -68,6 +61,10 @@ glue code thanks to powerful metaprogramming capabilities of Nim.
 - Micro: Included
 - Atom: [https://atom.io/packages/nim](https://atom.io/packages/nim)
 
+## What have been the major influences in the language's design?
+
+The language borrows heavily from (in order of impact): Modula 3, Delphi,
+Ada, C++, Python, Lisp, Oberon.
```

Comment:
```suggestion
Ada, C++, Python, Lisp, Oberon, D.
```

Reply:
But D wasn't an influence in Nim's design, believe it or not. Method call syntax, the effect system, the metaprogramming, "compile time function evaluation" was all influenced by papers, C++ or Lisp or ideas of my own. CTFE for example wasn't even a feature of its own, it's a prerequesite for Nim's imperative macro system.